.travis.yml

```diff
@@ -24,5 +24,5 @@ before_install:
   - git clone -b master https://$GITHUB_USER:$GITHUB_PWD@github.com/IBM-Swift/Testing-Credentials.git
 
 script:
-  - if [[ "$COMPONENT" == "server" ]]; then cd BluePic-Server && ./Package-Builder/build-package.sh -projectDir $TRAVIS_BUILD_DIR -credentialsDir $TRAVIS_BUILD_DIR/Testing-Credentials/BluePic/ci; fi
+  - if [[ "$COMPONENT" == "server" ]]; then cd BluePic-Server && ./Package-Builder/build-package.sh -projectDir $TRAVIS_BUILD_DIR -credentialsDir $TRAVIS_BUILD_DIR/Testing-Credentials/BluePic/development; fi
```

Comment:
Yeah, that's the plan. I didn't want to break anyone else's builds, since I had to change the format for cloud environment. I have a branch on testing-credentials ready to be merged when this gets the green light and I'll change this back then as well